loc/translations-import/vscode-powerplatform.zh-CN.xlf

```diff
@@ -48,22 +48,17 @@
       </trans-unit>
       <trans-unit id="pacCLI.authPanel.welcome.whenInteractiveSupported">
```

Comment:
Missing translations for `pacCLI.authPanel.welcome.whenInteractiveSupported` across all languages.

Reply:
Ignoring for now, Translation team will get the string in another PR